app/assets/stylesheets/show.scss

```diff
@@ -62,6 +62,10 @@
   }
 }
 
-span {
+span.voted {
+  padding: 10px;
+}
+
+span.nil_vote {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `nil_vote` should be written in lowercase with hyphens